app/services/katello/pulp3/migration.rb

```diff
@@ -3,20 +3,27 @@
 module Katello
   module Pulp3
     class Migration
-      attr_accessor :smart_proxy
+      attr_accessor :smart_proxy, :reimport_all
       GET_QUERY_ID_LENGTH = 90
 
-      REPOSITORY_TYPES = [
-        Katello::Repository::FILE_TYPE,
-        Katello::Repository::DOCKER_TYPE
-      ].freeze
-
       MUTABLE_CONTENT_TYPES = [
         Katello::DockerTag,
         Katello::Erratum
       ].freeze
 
-      def initialize(smart_proxy, repository_types = REPOSITORY_TYPES)
+      UNIFIED_CONTENT_TYPES = [
+        Katello::Erratum
+      ].freeze
+
+      def self.repository_types_for_migration
+        #we can migrate types that pulp3 supports, but are overridden to pulp2.  These are in 'migration mode'
+        overridden = (SETTINGS[:katello][:use_pulp_2_for_content_type] || {}).keys.select { |key| SETTINGS[:katello][:use_pulp_2_for_content_type][key] }
+        overridden.select { |type| SmartProxy.pulp_master.pulp3_repository_type_support?(type.to_s, false) }.map { |t| t.to_s }
```

Comment:
Looks like `master` snuck in to replace `primary` for the yum migration commits.